src/arns.lua

```diff
@@ -40,6 +39,18 @@ NameRegistry = NameRegistry or {
 --- @field target string|nil The address of the target of the reserved record
 --- @field endTimestamp number|nil The time at which the record is no longer reserved
 
+--- @alias ReturnedNameInitiator string -- The address of the initiator of the returned name
+
+--- @class ReturnedName
+--- @field name string The name of the returned record
+--- @field initiator ReturnedNameInitiator
+--- @field startTimestamp number The timestamp of when the record was returned
+
+--- @class ReturnedNameBuyRecordResult -- extends above
+--- @field initiator ReturnedNameInitiator
+--- @field rewardForProtocol number -- The reward for the protocol from the returned name purchase
```

Comment:
Nit: There's an alias for mIO

Reply:
[9ad13c4](https://github.com/ar-io/ar-io-network-process/pull/245/commits/9ad13c484a4e65e86136c2d5c4f8ea80f44849b8)